solution/backend/index_resources.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+import os
+
+
```

Comment:
this is only going to be called on experimental, dev, and val similar to seed data